runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -180,7 +181,7 @@ public void onNavigationButtonAdded(final NavigationButtonAdded event)
 			final NavigationButton navigationButton = event.getButton();
 			final PluginPanel pluginPanel = navigationButton.getPanel();
 			final boolean inTitle = !event.getButton().isTab() &&
```

Comment:
If you are using the withTitlebar var you created then SwingUtil.isCustomTitlePanePresent is no longer necessary